includes/polling/applications/backupninja.inc.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+// Polls backupninja statistics from script via SNMP
+use LibreNMS\RRD\RrdDefinition;
+
+$name = 'backupninja';
+$app_id = $app['app_id'];
+
```

Comment:
your json loader isn't as it should:

take a look into:
includes/polling/applications/seafile.inc.php
or
includes/polling/applications/puppet.inc.php
or
includes/polling/applications/redis.inc.php
...